lib/test/mdx_test.ml

```diff
@@ -118,7 +118,7 @@ let run_cram_tests ?syntax t ?root ppf temp_file pad tests =
         output;
       Cram.pp_exit_code ~pad ppf n)
     tests;
-  Block.pp_footer ?syntax ppf t
+  match syntax with Some Syntax.Mli -> () | _ -> Block.pp_footer ?syntax ppf t
```

Comment:
`Block.pp_footer` takes the `syntax` already, I think it should do the right thing. Perhaps that `pp_footer` inserts a newline is wrong ? `pp_contents` already control the newlines so it should be fixed there: https://github.com/Leonidas-from-XIV/mdx/pull/1

Reply:
I added this to be consistent with how this is done in the toplevel blocks but it isn't very nice as it basically checks twice for `Syntax.Mli`, once here to ignore it and once in `pp_footer` again where it does the `\n` dance. Your branch is much nicer, I merged it it! :+1: